lms/templates/instructor/instructor_dashboard_2/metrics.html

```diff
@@ -66,7 +66,6 @@ <h3 class="hd hd-3">${_("Grade Distribution Data")}</h3>
   %endfor
   <script>
     $(function () {
-      var firstLoad = true;
```

Comment:
Yes, but the previous system acted in this way and we didn't like it. That's why we overrode the default behaviour.